tests/cases/fourslash/importStatementCompletions1.ts

```diff
@@ -20,7 +20,7 @@
 //// [|import f/*4*/ =|]
 
 // @Filename: /index5.ts
-//// import f/*5*/ from "";
+//// [|import f/*5*/ from "";|]
```

Comment:
This test was wrong before because it had a range missing, but we weren't catching it because the expected `replacementSpan` was `undefined`.